components/landing/HeroBanner.vue

```diff
@@ -20,7 +20,7 @@
         </div>
       </h1>
       <div class="container is-fluid flex justify-center mt-[2rem]">
-        <LandingFarcasterBanner />
+        <LandingFarcasterMintNotificationBanner />
```

Comment:
Is it safe to remove the old component?

Reply:
According to the issue description, the old component will still be used in the future.

Reply:
yes

Reply:
meanwhile there is another one
https://warpcast.com/th0mas-art/0x47bfc921
- not sure if date range makes sense 


better to replace it fast with other ones, make the component as easily editable

Reply:
> In that case, can we put a date range? If out of range, will switch back auto without deployment

It's still not flexable to use date range. Let's leave it to the worker side. https://github.com/kodadot/private-workers/issues/118

Reply:
> meanwhile there is another one https://warpcast.com/th0mas-art/0x47bfc921
> 
> * not sure if date range makes sense
> 
> better to replace it fast with other ones, make the component as easily editable

Updated. It's easy to update the frontend config but need a release. For the long term we will update it from worker.

Reply:
Since the config is pretty simple, adding a JSON file to this repo is another workaround. Create a new file in `script/farcaster-live.json` and then fetch it with the raw GitHub URL. If 404/empty, return the old component.

For example, we can fetch this URL `https://raw.githubusercontent.com/kodadot/nft-gallery/main/script/posts.json` as a JSON. So, anyone can feel free to edit the configuration. It just needs to merge with the `main` branch. Simpler, no external dependency is required (workers/cf-kv), and no deployment is needed.

Reply:
> Since the config is pretty simple, adding a JSON file to this repo is another workaround. Create a new file in `script/farcaster-live.json` and then fetch it with the raw GitHub URL. If 404/empty, return the old component.
> 
> For example, we can fetch this URL `https://raw.githubusercontent.com/kodadot/nft-gallery/main/script/posts.json` as a JSON (I'm not sure about the CORS issue). So, anyone can feel free to edit the configuration. It just needs to be merged with the `main` branch. Simpler, no external dependency is required (workers/cf-kv), and no deployment is needed.

Thanks for providing the workaround. I give it a shot but you guess right. There is a CORS issue(at least on localhost).. So we will probably use the worker/cf-kv in this case in the future.
<img width="798" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/53b6358f-eebe-4974-92a1-a35dd25f09e8">

Reply:
Interesting, in my localhost, it looks fine. The code to fetch:

```js
const { data } = await useFetch(
  'https://raw.githubusercontent.com/kodadot/nft-gallery/main/script/posts.json',
)
```

Request origin from localhost:9090:

<img width="400" alt="Screenshot 2024-04-23 at 23 55 21" src="https://github.com/kodadot/nft-gallery/assets/734428/57333af4-75b4-40d9-8f52-1af7b32b0828">

Reply:
Make sure to request it using the `GET` method. I'm getting that error if the request is using a non-allowed method, such as `OPTIONS`

Reply:
Thanks. I use `$fetch` instead of native fetch, then it works without CORS error.